### PR TITLE
Feature/instructions page

### PR DESCRIPTION
**Pull request by neozhixuan**
Added the basic functionalities of the instructions page, displaying relevant information according to the user's selections.

### CODE REVIEW

File: client/spa-pages/components/InstructionsPage.tsx [hunk 1]
```diff
@@ -1,32 +1,83 @@
...
+			if (instructionItem.step3 !== undefined) {
+				contents.push(instructionItem.step3);
+			}
+			return {
+				title: instructionItem.name,
+				method: "",
+				contents,
+			};
+		})
+		.map((instrItem) => {
```
Comment (ryhuz) [inline client/spa-pages/components/InstructionsPage.tsx:43]: can this map be merged with the previous map?

File: client/components/instructions/InstructionsAccordion/index.tsx [hunk 2]
```diff
@@ -48,13 +47,13 @@ export const AccordionComp = ({ items }: AccordionProps) => {
 						</AccordionButton>
 					</h2>
 					<AccordionPanel>
-						<Center>
+						{/* <Center>
```
Comment (ryhuz) [inline client/components/instructions/InstructionsAccordion/index.tsx:50]: remove if not needed

**Review by ryhuz — CHANGES_REQUESTED** [review]
additionally, merge master to get the latest updates on the context folder organisation

since `instructions` is part of sheety functionality, it can go into that folder

File: client/spa-pages/components/InstructionsPage.tsx (hunk 1 above)
Comment (neozhixuan) [reply client/spa-pages/components/InstructionsPage.tsx:43]: Ah yes, managed to do so

**Review by ryhuz — APPROVED** [review]

**Pull request merged**